README.md

```diff
@@ -4,10 +4,41 @@ A 'package manager' for Glorious Eggroll's custom proton build.
 
 ## Installation
 
+There are currently a couple of ways to install `humble-lumpia`.
+
+**NOTE:** Installing `humble-lumpia` and its dependencies is not sufficient to 
+run the versions of `proton-ge-custom` that `humble-lumpia` will download. Be 
+sure to reference the [README](https://github.com/GloriousEggroll/proton-ge-custom/)
+in `proton-ge-custom` to determine what other dependencies your system requires.
+
+### AUR Package Installation
```

Comment:
Nit: `###From the AUR` here would better match the next section (`###From Source`)

Reply:
I updated the wording as you suggested. I agree that it looks better with consistent wording.